tools/config.py

```diff
@@ -17,7 +17,7 @@
 IMAGE_FORMAT = 'vmdk'
 IMAGE_DESTINATION_URI = f'gs://meilisearch-image/meilisearch-{MEILI_CLOUD_SCRIPTS_VERSION_TAG}-{DEBIAN_BASE_IMAGE_FAMILY}.{IMAGE_FORMAT}'
 IMAGE_DESTINATION_BUCKET_NAME = 'meilisearch-image'
-SERVICE_ACCOUNT_EMAIL = '591812945139-compute@developer.gserviceaccount.com'
```

Comment:
So indeed we had to change something in the permissions 👀

Reply:
No, in fact, it was the typo that caused the error. We changed the service account because Samuel would rather use this one. And yes, it works well, I ran it locally